assessment/collectors/app_migration_analysis.go

```diff
@@ -30,28 +28,38 @@ import (
 	assessment "github.com/GoogleCloudPlatform/spanner-migration-tool/assessment/collectors/embeddings"
 	parser "github.com/GoogleCloudPlatform/spanner-migration-tool/assessment/collectors/parser"
 	dependencyAnalyzer "github.com/GoogleCloudPlatform/spanner-migration-tool/assessment/collectors/project_analyzer"
+	. "github.com/GoogleCloudPlatform/spanner-migration-tool/assessment/utils"
 	"github.com/GoogleCloudPlatform/spanner-migration-tool/logger"
+	"go.uber.org/zap"
 )
 
 // MigrationSummarizer holds the LLM models and example databases
 type MigrationSummarizer struct {
-	projectID          string
-	location           string
-	client             *genai.Client
-	modelPro           *genai.GenerativeModel
-	modelFlash         *genai.GenerativeModel
-	conceptExampleDB   *assessment.MysqlConceptDb
-	dependencyAnalyzer dependencyAnalyzer.DependencyAnalyzer
-	sourceSchema       string
-	targetSchema       string
+	projectID                     string
+	location                      string
```

Comment:
what do you need the location for ?

Reply:
It is required for the endpoint of creating embeddings.